app/views/application/_head.html.erb

```diff
@@ -40,7 +40,7 @@
 
 <%= render 'analytics', application: 'docs', title: "Docs / #{content_for(:page_title)}", logged_in: probably_authenticated? %>
 
-<% if ENV.fetch("ROBOTS_NO_INDEX") { "false" } == "true" %>
+<% if ENV.fetch("ROBOTS_NO_INDEX") { "false" } == "true" || request.base_url == "https://badge.buildkite.com" %>
```

Comment:
hi @hannahcancode as a thought, should we address the `https://badge.buildbox.io` domain as well?

Currently https://badge.buildbox.io also lands the user on the Docs site. It's also indexed in Google.

Acknowledging this PR specifically addresses the `badge.` subdomain. Over to you to decide the scope of this PR.

Reply:
Oh good callout @olyism, I didn't even know about badge.buildbox.io. I suspect it's not as big an issue because it doesn't pollute buildkite.com in the google index, but seems sensible to get rid of it too. I'll push up something for that.